src/controllers/logging/ChannelLog.cpp

```diff
@@ -9,7 +9,8 @@ ChannelLog::ChannelLog(QString channelName)
 
 QString ChannelLog::channelName() const
 {
-    return this->channelName_;
+    QString lowercaseChannelName = this->channelName_.toLower();
+    return lowercaseChannelName;
```

Comment:
nit: You don't have to make a temporary variable here, you can just return this->channelName_.toLower(); since [QString::toLower](https://doc.qt.io/qt-5/qstring.html#toLower) returns a copy of the string

Reply:
Okay, thank you for the suggestion. Its an improvement over the current code.  Should I 1) leave the current code as is, 2) commit the line you suggested, or 3) wait for you to do a "suggested change" within the file?

Reply:
I decided to just do a commit since I saw it was a "requested change".

Reply:
That's the right option in my book, thank you!